<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Amar" ignored="0" name="Test-Runner-2" tests="1" failures="1" timestamp="2021-04-26T13:18:25 IST" time="54.178" errors="0">
  <testcase name="createcontact" time="31.988" classname="com.vtiger.tc.CreateCOntact_Org">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.vtiger.tc.CreateCOntact_Org.createcontact(CreateCOntact_Org.java:64)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- createcontact -->
</testsuite> <!-- Test-Runner-2 -->
